library/common/jni/jni_interface.cc

```diff
@@ -116,6 +116,8 @@ static void pass_headers(JNIEnv* env, envoy_headers headers, jobject j_context)
 }
 
 // Platform callback implementation
+// These methods call jvm methods which means the local references created will not be
+// released automatically. Manual book keeping is required for these methods.
```

Comment:
```suggestion
// released automatically. Manual bookkeeping is required for these methods.
```

Trivia: "bookkeeping" is the only English word with three pairs of double letters in a row.